### PR TITLE
Update tower binary

### PR DESCRIPTION
**Pull request by fevac**
### Changed

- Use general tower binary instead of specifying it as part of the pipeline

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b tower_binary -a

  bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-servers-stage.sh tower_binary

    ```

### How to test

- [x] `cg workflow rnafusion run -d CASE`. Make sure that tower binary path is specified

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Review by islean — APPROVED** [review]
🔥

File: cg/meta/workflow/rnafusion.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def __init__(
         self.profile: str = config.rnafusion.profile
         self.conda_env: str = config.rnafusion.conda_env
         self.conda_binary: str = config.rnafusion.conda_binary
-        self.tower_binary_path: str = config.rnafusion.tower_binary_path
```
Comment (islean) [inline cg/meta/workflow/rnafusion.py:49]: Do you need to do the same for raredisease?

**Review by islean — APPROVED** [review]
Looks good! Just one question about the raredisease change.

File: cg/meta/workflow/rnafusion.py (hunk 1 above)
Comment (fevac) [reply cg/meta/workflow/rnafusion.py:49]: that part is not implemented yet on rarediseases

Comment (islean) [reply cg/meta/workflow/rnafusion.py:49]: Then that makes sense!

**Comment by fevac** [comment]
tests are passing

**Review by ivadym — APPROVED** [review]

**Comment by fevac** [comment]
- deployed to stage
image

-deployed to prod (next version)
image

**Pull request merged**